src/main/java/com/mael/ttt/Game.java

```diff
@@ -16,42 +15,24 @@
     private String playerMark;
     private String opponentMark;
 
-    public Game(Board board, UserInterface gameUI) {
+    public Game(Board board, UserInterface gameUI, List<Player> players) {
         this.board   = board;
         this.gameUI  = gameUI;
+        player1      = players.get(0);
```

Comment:
Careful with the `.get(0)` and `.get(1)` calls.
From the outside, the constructor takes a `List<Player>` which can be 1, 2,3,...n players.
Inside, you actually reduce that number down to exactly 2. I wish Java had tuple or pair, but short of that I prefer having 2 parameters in the constructor. It makes it obvious that needs exactly two players, not more or less :)